File: src/main/java/com/linebot/model/train/TrainDelay.java
package com.linebot.model.train;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.extern.slf4j.Slf4j;

/**
 * ほんとは"TrainDelayResponse"にしたかった。
 */
@Slf4j
@Data
@NoArgsConstructor
@AllArgsConstructor
public class TrainDelay {
	private String name;
	private String company;
	private String lastupdate_gmt;
	private String source;
}

File: src/main/kotlin/com/linebot/model/train/TrainDelay.kt
package com.linebot.model.train

/**
 * ほんとは"TrainDelayResponse"にしたかった。
 */
data class TrainDelay(
    var name: String? = null,
    var company: String? = null,
    var lastupdate_gmt: String? = null,
    var source: String? = null
)
